build/bump-version.js

```diff
@@ -0,0 +1,138 @@
+const fs = require('fs');
+const os = require('os');
+
+const packageJsonLoc = './frontend/package.json';
+const csprojLoc = './backend/api/Pims.Api.csproj';
+
+/**
+ * USAGE:
+ *
+ * node ./build/bump-version.js [<new-version> | major | minor | patch | is | build] [--apply] [--release-version]
```

Comment:
I'd be tempted to put the purpose of the included options in this section, ie: what does release-version do?
I know because I read the code, but its nice to have the expected functionality documented here.

Reply:
I did put a comment on the next line saying what "--release-number" does LOL. I can make the note more prominent. Also maybe change the flag to "--print-version" might indicate what it does a bit better?